tests/services/assignment_service_test.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+from unittest.mock import MagicMock
+from sqlalchemy.orm import Session
+from app.services import AssignmentService
+from app.models import AssignmentModel
+from app.core.exceptions import AssignmentNotFoundException
+from ..data.database.assignment import assignment_data
+
+class TestAssignmentService(unittest.IsolatedAsyncioTestCase):
```

Comment:
This file should have the same name as the class

Reply:
The file name is `assignment_service.py`, do we want to start prefixing with "test" followed by the class name?
Ex: `test_assignment_service.py` - rather than suffix which I have currently

Reply:
Considering what Griffin said the other day about test files always starting with "test", I would say yes. They don't have to always say test_<filename>, it could be test_<something general> though.